packages/orm/src/index.ts

```diff
@@ -140,6 +140,8 @@ export function setOpts<T extends Entity>(
           const allowDelete = !field.otherMetadata().fields["delete"];
           const allowRemove = !field.otherMetadata().fields["remove"];
 
+          const isSoftDeletable = !!field.otherMetadata().allFields["deletedAt"];
```

Comment:
We support configurable deleted at column names, so maybe use:

```
  const { entity, createdAt, updatedAt, deletedAt } = meta;
```

To get the field name. Might have to do like `getBaseMeta(field.otherMetadata()).deletedAt`

If that's defined, it will tell you: a) that soft delete is enabled on this entity, and b) what the field name is to use later in the `.set` call.